### ISSUE TITLE
Improve CostEstimationHandler

### ISSUE THREAD

**Issue by konradkonrad** [raiden-network/raiden-wizard#133]
For the swapping frontend, we want to be able to asynchronously query exchange rates.

There already is an endpoint for this: https://github.com/raiden-network/raiden-wizard/blob/30b9e89d811ac4cc7b06c75621be720d071b8a0b/raiden_installer/web.py#L557

We should change the [CostEstimationHandler](https://github.com/raiden-network/raiden-wizard/blob/30b9e89d811ac4cc7b06c75621be720d071b8a0b/raiden_installer/web.py#L514), so it responds to `POST` requests with the following parameters:
```
{
  "exchange": "kyber|uniswap",
  "currency": "DAI|RDN",
  "target_amount": <number>
}
```
with a response like this:
```
{
  "exchange": "kyber|uniswap",
  "currency": "DAI|RDN",
  "target_amount": <number>,
  "estimated_cost": <number>,
  "utc_seconds": <number>
}
```

So, for a complete example:
```
POST /api/cost-estimation/<configuration-file>
{
  "exchange": "kyber",
  "currency": "RDN",
  "target_amount": 10
}
# Response
{
  "exchange": "kyber",
  "currency": "RDN",
  "target_amount": 10,
  "estimated_cost": "0.02353456221433",
  "utc_seconds": 1588165525
}
```

**Comment by manuelwedler** [comment]
Looks great! Maybe add the estimated costs as wei and formatted, as it is now. For example:
```
{
   "as_wei": 9397348491152692,
   "formatted": "0.00940 ETH"
}
```

**Comment by agatsoh** [comment]
Code returns 
```
{
  "exchange": "kyber",
  "currency": "RDN",
  "target_amount": 10,
  "as_wei": 14426829418305688,
  "formatted": "0.0144 ETH",
  "utc_seconds": 1588245104
}
```
`estimated_cost` would be the same as the `as_wei` and `formatted`

**Comment by fredo** [comment]
closed due to #138
